src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -114,7 +193,10 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
         </TableControlInnerRow>
       </TableControlOuterRow>
       <TableContainer hide={showSkeleton}>
-        <TasksTable columns={columns} data={get(data, "patchTasks", [])} />
+        <TasksTable
+          columns={columnsTemplate}
+          data={get(data, "patchTasks", [])}
```

Comment:
this can be `data?.patchTasks ?? []`

Reply:
I'd have to assert that the empty array is an empty array of PatchTasks which I think is messier than what's here